src/videoTrackingHTML5.js

```diff
@@ -237,10 +258,20 @@
         mediaElement.addEventListener("ended", processEndedMediaEvent);
         mediaElement.addEventListener("volumechange", processVolumeChangeMediaEvent);
     }
+
+    function addListenersForEachItem(mediaElements){
+        mediaElements.forEach((mediaElement) => addMediaTrackingListeners(mediaElement));
+        window.addEventListener("beforeunload", handleTabUnloadDuringVideoPlay);
+    }
     
     let mediaElements = document.querySelectorAll(videoElementSelector);
-    
-    mediaElements.forEach((mediaElement) => addMediaTrackingListeners(mediaElement));
-    window.addEventListener("beforeunload", handleTabUnloadDuringVideoPlay);
+    if (mediaElements[0] != null && mediaElements[0] != undefined) {
```

Comment:
You want to check if any element is returned? So you will have empty array returned. Better option will be:
```suggestion
    if (mediaElements.length) {
```

Anyway please always use strict compare operators (=== or !==).

Reply:
Thanks, I actually forgot about the strict operators (must have mixed up with python/java). I commited the change.